common/views/components/PageLayout/PageLayout.tsx

```diff
@@ -48,6 +48,7 @@ export type Props = {
   image?: ImageType;
   rssUrl?: string;
   children: ReactNode;
+  hideGlobalAlert?: boolean;
```

Comment:
This means we don't show the global alert on exhibition guides (e.g. "We will be closed on Monday 19 September for the funeral of Her Majesty Queen Elizabeth II.")

We haven't discussed this, but it feels like the right thing to do – we've tried to minimise the amount of chrome on these pages.